cpp/src/plasma/events.h

```diff
@@ -83,11 +83,14 @@ class EventLoop {
   /// @return The ae.c error code. TODO(pcm): needs to be standardized
   int remove_timer(int64_t timer_id);
 
-  /// Run the event loop.
+  /// \brief Run the event loop.
   ///
   /// @return Void.
   void run();
```

Comment:
Sounds good. I made a clean sweep over the rest of the EventLoop methods for consistency